app/controllers/games_controller.rb

```diff
@@ -34,11 +34,38 @@ def update
 
   private
 
-  helper_method :current_game
+  helper_method :current_game, :place_piece_td
   def current_game
     @current_game ||= Game.find(params[:id])
   end
 
+  def board_display_piece_query(row, column)
+    current_game.pieces.find do |f|
+      f["x_position"] == column && f["y_position"] == row
+    end
+  end
```

Comment:
this queries the database to see if there is a piece in the current board position